deployment/kube/stage/frontend.yaml

```diff
@@ -43,6 +54,13 @@ spec:
         ports:
         - name: http-server
           containerPort: 9091
+        resources:
+         limits:
+           cpu: 600m
+           memory: 1Gi
```

Comment:
Why limit to so little memory? The nodes have 3.75 GB available, and only one imageserver pod should be running on each node.

Reply:
OK, sounds good.